cypress/e2e/2-locators-and-actions/second.cy.js

```diff
@@ -0,0 +1,16 @@
+describe('Check form "Enter name"', ()=>{
+    context('page modal-overlays/dialog', ()=>{
+
+it ('Check form "Enter name" on page modal-overlays/dialog(Return Result From Dialog)', ()=>{
+cy.visit('pages/modal-overlays/dialog');
+cy.get('.appearance-filled').last().click()
+cy.get('input').parent().should('be.visible')
+cy.contains('Enter your name').should('have.text', 'Enter your name')
```

Comment:
```suggestion
cy.contains('Enter your name').should('be.visible')
```
you can leave as visible